pages/features/cache.mdx

```diff
@@ -0,0 +1,110 @@
+# FlakeHub Cache
+
+*FlakeHub Cache* is a [Nix binary cache][cache] that works in conjunction with [FlakeHub][home].
+It has two main features that distinguish it from other Nix caching services:
+
+1. It enables you to [control access](#access-control) at the [flake][flakes] level.
+1. It uses a flexible and convenient [authentication](#authentication) model that doesn't require juggling access keys.
+
+FlakeHub Cache is currently in **private beta** but you can sign up for the beta here and we'll add you as soon as we're ready:
+
+<SignupForm />
+
+## Access control
+
+Nix caching solutions based on [nix-serve] serve an entire [Nix store][store] as a single monolithic cache and make all store paths in that cache available to anyone with a public key.
+With FlakeHub Cache, on the contrary, you can grant or deny read access to specific users **at the [flake][flakes] level**, which affords substantially more granular access control.
+
+Let's say that you're an admin for the org `omnicorp`.
+You could manage access to your org's flakes in such a way that, for example:
+
+* Only some trusted users would be allowed to pull store paths associated with the flake `super-duper-secure`.
+* Anyone in your org can pull store paths associated with the `dev-environments` flake.
+
+## Authentication model
+
+FlakeHub Cache authentication is based on [JSON Web Tokens][jwt] (JWTs or "jots").
+We currently use [GitHub] as our JWT authentication provider but will be adding support for [GitLab] in the near future.
```

Comment:
Should update this to say that we support GitLab.

Reply:
Right now we kind of support GitLab *for LastPass* but I'm not sure it's in a place where we want to advertise that